google/resource_compute_disk_test.go

```diff
@@ -292,7 +292,7 @@ func testAccComputeDisk_deleteDetach(instanceName, diskName string) string {
 	return fmt.Sprintf(`
 resource "google_compute_disk" "foo" {
 	name = "%s"
-	image = "debian-8"
+	image = "debian-8-jessie-v20170523"
```

Comment:
> we should discourage users from simply using the debian-8 shorthand going forward.

Given that these tests fail, I think we've inadvertently already started discouraging users from using it by breaking them 😞 

Mind filing an issue to make our lack of support for that shorthand more explicit? I think there are a few places in the code where that needs to be changed.

Once that's done, feel free to merge this when you're ready 👍